go/bundle/cmd/gen-bundle/main.go

```diff
@@ -101,6 +101,15 @@ func fromHar(harPath string) error {
 			return fmt.Errorf("Failed to extract body from response content for the request %q. err: %v", e.Request.URL, err)
 		}
 
+		if e.Request.Method != "GET" {
+			log.Printf("Dropping the entry: non-GET request method (%s)", e.Request.Method)
+			continue
+		}
+		if http.StatusText(e.Response.Status) == "" {
```

Comment:
I'm not sure on this change. Would you provide more context?

Reply:
Ah, how about simple check `Status != 0`?

Reply:
or >=100 <=999

Reply:
Changed to check that status code is within [100, 999].